### PR TITLE
Order info

### PR DESCRIPTION
**Pull request by ileenshaar**

### CODE REVIEW

File: src/components/order-info/order-info.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    display: "flex",
+    justifyContent: "space-between",
+    marginBottom: "12px",
+    fontSize: "16px",
+    fontWeight: "500",
+  };
+  return (
+    <>
+      <h1 style={{ fontSize: "20px", marginBottom: "8px" }}>Order Details</h1>
+      <hr style={{ marginBottom: "35px" }} />
```
Comment (mahmoud-alragabi) [inline src/components/order-info/order-info.jsx]: move the styles into css

File: src/components/order-info/order-info.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          style={{
+            ...tableContainer,
+            ...(index === orderInfo.length - 1 ? { fontWeight: "bold" } : {}),
+          }}
+        >
+          <div
+            style={{
+              color: "#626262",
+              ...(index === orderInfo.length - 1 ? { color: "#171520" } : {}),
+            }}
```
Comment (mahmoud-alragabi) [inline src/components/order-info/order-info.jsx]: again move styles to css

File: src/components/order-info/order-info.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          >
+            {detail}
+          </div>
+          <div style={{ color: "#171520" }}>{amount}</div>
```
Comment (mahmoud-alragabi) [inline src/components/order-info/order-info.jsx]: here as well

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

File: src/components/order-info/order-info.jsx [hunk 4]
```diff
@@ -1,34 +1,28 @@
...
 
       {orderInfo.map(([detail, amount], index) => (
         <div
           key={index}
-          style={{
-            ...tableContainer,
-            ...(index === orderInfo.length - 1 ? { fontWeight: "bold" } : {}),
-          }}
+          className={`${styles.tableContainer} ${
+            index === orderInfo.length - 1 ? styles.bold : ""
```
Comment (mahmoud-alragabi) [inline src/components/order-info/order-info.jsx]: this operation shouldn't be online should be saved to the top of the component then use that variable here

File: src/components/order-info/order-info.jsx [hunk 5]
```diff
@@ -1,34 +1,28 @@
...
         >
           <div
-            style={{
-              color: "#626262",
-              ...(index === orderInfo.length - 1 ? { color: "#171520" } : {}),
-            }}
+            className={
+              index === orderInfo.length - 1
+                ? styles.darkFont
+                : styles.lightFont
```
Comment (mahmoud-alragabi) [inline src/components/order-info/order-info.jsx]: here as well saved into a variable then used here, shouldn't be doing this inline

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by mahmoud-alragabi — APPROVED** [review]

**Review by ferassmohana — APPROVED** [review]

**Pull request merged**